.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  target-branch: dev
+  commit-message:
+    prefix: "[Dependabot]"
+  labels:
+      - Dependencies
```

Comment:
I think this might produce errors because label Dependencies does not exist.

Reply:
Label is now created.